app/routes/__init__.py

```diff
@@ -10,7 +10,7 @@
 from ..utils.rw_api import who_am_i
 
 DATASET_REGEX = r"^[a-z][a-z0-9_-]{2,}$"
-VERSION_REGEX = r"^v\d{1,8}\.?\d{1,3}\.?\d{1,3}$|^latest$"
+VERSION_REGEX = r"^v\d{1,8}(\.\d{1,3}){0,2}?$|^latest$"
```

Comment:
Is there a reason we actually need to enforce version names like this? E.g. for data updates, it'd be nice to have a version like v2020Q1. Why not just allow any alphanumeric character or period with a size limit?

Reply:
Good question.

I prefer to have versioning schemas which are sortable so that the release order is clear. If allowing any alphanumeric character this could grow quickly out of control. Instead you could do `v2020.1`

Reply:
Ah ok, that makes sense to me.